CONTRIBUTING.md

```diff
@@ -123,68 +125,92 @@ accepted.
-and produce up to date documentation and minimise bugs. A few gotchas while you
-contribute might be:
-
-- the readme files in the components are auto-generated. You won't need to update
-  them. A big chunk of the documentation and prop-type validation is happening on the `doc.js` files.
-- code coverage and unit-testing is an important process of the development. A
-  pre-commit hook exists that runs jest tests. To manually run them you should run
-  `yarn test`. If you need to update snapshots then run `yarn test-update`
-- we are actively working on providing a seamless Typescript experience.
-  Don't forget to update the corresponding `index.d.ts` files.
-- for code syntax alignment on your pull request, use 'prettier'.
-- pull requests with code should include tests that validate your change.
-
-We review issues and pull requests on a weekly basis (sometimes more frequent). If you feel we missed yours don't hesitate to ping us on slack!
-
-## Contributing to the documentation
+1. fork the `grommet` repository
+1. clone it `git clone https://github.com/<your-username>/grommet.git`
+1. install dependencies using: `yarn install`
+
+The components code lives in `src/js/components`. The structure of the
+project is a bit particular since it is using lots of internal tooling to try to
+produce up-to-date documentation and minimise bugs. A few gotchas you may run
+into while contributing could include:
+
+- The read-me files in the components are auto-generated. You won’t need to
+  update them. A big chunk of the documentation and prop-type validation is
+  happening via the `doc.js` files.
+- Code coverage and unit-testing is an important process of development.
+  A pre-commit hook exists which runs the test suite and aborts the commit if
+  any fail. To manually run tests, you should run `yarn test`. If you need to
+  update snapshots then run `yarn test-update`.
+- We are actively working on providing a seamless TypeScript experience. Don’t
+  forget to update corresponding `index.d.ts` files.
+- For code syntax alignment in your pull request, use [prettier].
+- Pull requests with code should include tests that validate the changes.
+
+We review issues and pull requests on a weekly basis (sometimes more
+frequently). If you feel we missed yours don’t hesitate to ping us on
+[Slack][slack community]!
+
+## Contributing to the Documentation
 
 Grommet uses an internal tool for most of its documentation. If you are looking
-to modify component to documentation then you only need to have a look at the
+to modify component documentation then you only need to have a look at the
 `doc.js` files.
 
-These files are used to generate the documentation on the grommet website. That
-code lives at the [grommet-site](https://github.com/grommet/grommet-site) repository.
+These files are used to generate the documentation on the Grommet website. That
+code lives in the [grommet-site] repository.
```

Comment:
why did we remove the link?

Reply:
See response about reference-style vs. inline-style links in Markdown.